src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -120,7 +120,6 @@ public function implementedEvents()
         return [
             'Controller.startup' => 'startup',
             'Controller.beforeRender' => 'beforeRender',
-            'Controller.beforeRedirect' => 'beforeRedirect',
```

Comment:
For me those changes look rather BC breaking, at least the method must stay, so people extending will not suddenly have their beforeRedirect not executed anymore IMO.